pkgs/desktops/gnome/extensions/systemd-manager/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "gnome-shell-extension-systemd-manager";
+  version = "16";
+
+  src = fetchFromGitHub {
+    owner = "hardpixel";
+    repo = "systemd-manager";
+    rev = "v${version}";
+    hash = "sha256-JecSIRj582jJWdrCQYBWFRkIhosxRhD3BxSAy8/0nVw=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/gnome-shell/extensions/
+    mv systemd-manager@hardpixel.eu $out/share/gnome-shell/extensions/
+  '';
+
+  passthru = {
+    extensionUuid = "systemd-manager@hardpixel.eu";
+    extensionPortalSlug = "systemd-manager";
+    updateScript = nix-update-script { };
```

Comment:
Is that really needed? I think both `nix-update` and `nixpkgs-update` will be able to update this package even without this - there is nothing non-trivial with the GitHub project - there are both releases and tags there.